docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -6,24 +6,27 @@ Delta Lake connector
 
   <img src="../_static/img/delta-lake.png" class="connector-logo">
 
-The Delta Lake connector allows querying data stored in `Delta Lake
+The Delta Lake connector allows querying data stored in the `Delta Lake
 <https://delta.io>`_ format, including `Databricks Delta Lake
-<https://docs.databricks.com/delta/index.html>`_. It can natively read the Delta
-transaction log, and thus detect when external systems change data.
+<https://docs.databricks.com/delta/index.html>`_. The connector can natively
+read the Delta transaction log and thus detect when external systems change
```

Comment:
Please check if its called the "Delta transaction log" or the "Delta Lake transaction log"

Reply:
Looks like it should be Delta Lake transaction log https://www.databricks.com/blog/2019/08/21/diving-into-delta-lake-unpacking-the-transaction-log.html

Reply:
My only original change was to go from the blind subject reference "It" to "The connector".